ephios/core/models/users.py

```diff
@@ -386,20 +387,14 @@ class Meta:
 )
 
 
-class Consequence(Model):
+@dont_log  # as we log the specific models
+class AbstractConsequence(PolymorphicModel):
```

Comment:
Currently, the code is not robust against consequence handlers being uninstalled (try removing the youth warden plugin). I think it should be, similar to missing structures and signup flow, it shouldn't crash the application. Maybe setting a custom `_default_manager` would be a simple solution?